maintainers/scripts/nix-guix-correlate.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python2
+
+'''
+A simple script to correlate the JSON package data from Nix and Guix and
+compare versions for packages that are the same in both.
+
+Written by Armijn Hemel -- same license as whatever nixpkgs uses
+'''
+
+import os, sys, json, httplib, gzip, StringIO
+
+## Guix exports all available packages as JSON
+## https://www.gnu.org/software/guix/packages/packages.json
+httpcon = httplib.HTTPSConnection('www.gnu.org')
+httpcon.request('GET', '/software/guix/packages/packages.json')
+res = httpcon.getresponse()
+if res.status != 200:
+	print >>sys.stderr, "Cannot grab GUIX JSON file"
+	sys.stderr.flush()
+	sys.exit(1)
+
+guixdata = res.read()
+
+'''
+## in case the data is cached -- TODO
+guixjsonfilename = '/tmp/packages-guix.json'
+guixfile = open(guixjsonfilename, 'rb')
+guixdata = guixfile.read()
+guixfile.close()
+'''
+
+try:
+	guixjson = json.loads(guixdata)
+except:
+	print >>sys.stderr, "Invalid GUIX JSON data"
+	sys.exit(1)
+
+## NixOS exports the available packages in the stable channel
+## (for example: 17.03) as JSON
+## http://nixos.org/nixpkgs/packages.json.gz
+httpcon = httplib.HTTPConnection('nixos.org')
```

Comment:
```suggestion
httpcon = httplib.HTTPSConnection('nixos.org')
```

Reply:
Oh, duh. I probably should be rewriting the script anyway to Python 3